src/content/getting-started/cloud-provider-accounts/vmware-vsphere/_index.md

```diff
@@ -53,25 +55,25 @@ The role must be assigned to the following objects:
 
 If you want to leverage failure domains at the host level where a group of hosts is a failure domain (stretched cluster, racks, PDU distribution...), CAPV will need to work with Anti-Affinity rules.
 
-As a result the user requires the following permissions: `Host > Edit > Modifiy cluster`
+As a result the user requires the following permissions: `Host > Edit > Modify cluster`
 
 ## Networking
 
-DHCP service must be enabled in the primary VM Network to assign IP addresses to the network interfaces of the nodes.
+The DHCP service must be enabled in the primary VM Network to assign IP addresses to the network interfaces of the nodes.
 
 Because the base implementation of CAPV includes a layer-2 load balancer, some customers may prefer to use NSX ALB, especially if they already have it in the environment. As a result, the implementation may vary across customers.
 
 {{< tabs >}}
 {{< tab id="flags-kubevip" title="kube-vip">}}
-Since vSphere has no concept of load balancer out of the box, CAPV ships with [kube-vip](../../../advanced/vsphere-kubevip/index.md), a layer-2 load balancer that works with ARP requests. By default, `kube-vip` only handles the Kubernetes API access but at Giant Swarm, we also deploy the `kube-vip` cloud provider to offer the capability to create services of type load balancer.
+Since vSphere has no concept of load balancers out of the box, CAPV ships with [kube-vip]({{< relref "/advanced/vsphere-kubevipt" >}}), a layer-2 load balancer that works with ARP requests. By default, `kube-vip` only handles the Kubernetes API access but at Giant Swarm, we also deploy the `kube-vip` cloud provider to offer the capability to create services of type load balancer.
```

Comment:
```suggestion
Since vSphere has no concept of load balancers out of the box, CAPV ships with [kube-vip]({{< relref "/advanced/vsphere-kubevip" >}}), a layer-2 load balancer that works with ARP requests. By default, `kube-vip` only handles the Kubernetes API access but at Giant Swarm, we also deploy the `kube-vip` cloud provider to offer the capability to create services of type load balancer.
```

You can run `make dev` to render changes and thereby check the build success. Invalid links lead to hard errors.